R/simulate.R

```diff
@@ -134,18 +136,19 @@ simulate_gillespie <- function(r = 0.2,
 #' @family simulate
 #' @export
 simulate_secondary <- function(linelist, dist = rlnorm, ...) {
+  delay <- ptime <- stime <- NULL
```

Comment:
I'm not sure why this is necessary.

Reply:
Yes, it's silly, but done to prevent linting problems with `data.table`. See: https://cran.r-project.org/web/packages/data.table/vignettes/datatable-importing.html (Dealing with "undefined global functions or variables")

Reply:
(its not just `data.table` its R package checks more generally)

Reply:
An alternative is to define them everywhere as global variables in utils which can be a bit less messy.